pyqg/model.py

```diff
@@ -651,7 +651,7 @@ def _increment_diagnostics(self):
                 if self.diagnostics[dname]['count']==0:
                     self.diagnostics[dname]['value'] = res
                 else:
-                    self.diagnostics[dname]['value'] += res
+                    self.diagnostics[dname]['value'] = self.diagnostics[dname]['value'] + res
```

Comment:
Isn't this the exact same thing?

Reply:
I was surprised that they turned out differently as well. As an example, I created two Binder environments: [the first one](https://mybinder.org/v2/gh/tlhsieh/pyqg/87152809b6ce7c90ade48c1cf29f75fa8ad3d95a) built from the master pyqg source ([yml here](https://github.com/tlhsieh/pyqg/blob/binder_github/environment.yml)), and [the other one](https://mybinder.org/v2/gh/tlhsieh/pyqg/aa00057cfb98c8ba662bcf58c499c2e9d671903c) built with this current modification ([yml here](https://github.com/tlhsieh/pyqg/blob/binder_tlh/environment.yml)). The diagnostic tool no longer returns the last snapshot.